start/pom.xml

```diff
@@ -43,19 +43,14 @@
       <artifactId>commons-vfs2</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-client</artifactId>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
```

Comment:
Looks like this one might have been a bug we probably should have fixed anyway. We shouldn't have been relying on this being brought in transitively.

Reply:
Yep. There was a fix done in master by mwalch cleaning up most of this crap using the maven-dependency-analyzer feature.

Reply:
Ah, cool. I thought that had been applied to all branches, but I guess not.